docpages/building/linux.md

```diff
@@ -37,12 +37,27 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
+If you are on Arch Linux you can add `/usr/local` to your enviorment variable:
+
+```bash
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH
+```
+
+OR if you dont want to change your enviorment you can use add these flags while compiling.
+
+```bash
+g++ -std=c++17 mydppbot.cpp -o dppbot -I/usr/local/include -L/usr/local/lib -ldpp
```

Comment:
This should be `g++ -std=c++17 -L/usr/local/lib -I/usr/local/include -ldpp mydppbot.cpp -o dppbot`

Reply:
nope -ldpp always goes on the end remember, otherwise it breaks on osx :)

Reply:
otherway round, -ldpp needs to be at the end for linux otherwise it breaks

Reply:
ok doing it